packages/devtools_app/lib/src/screens/vm_developer/object_inspector_view.dart

```diff
@@ -21,31 +21,44 @@ class ObjectInspectorView extends VMDeveloperView {
           title: 'Objects',
           icon: Icons.data_object,
         );
-  static const id = 'inspector-view';
+  static const id = 'object-inspector-view';
 
-  final programExplorerController = ProgramExplorerController();
+  @override
+  bool get showIsolateSelector => true;
 
+  @override
+  Widget build(BuildContext context) => _ObjectInspectorView();
+}
+
+class _ObjectInspectorView extends StatefulWidget {
   final controller = ObjectInspectorViewController();
```

Comment:
this controller should be initialized below in the State class's initState method.